rules/linux/auditd/lnx_auditd_disable_aslr_protection.yml

```
@@ -21,11 +22,14 @@ logsource:
     product: linux
     service: auditd
 detection:
-    selection:
+    selection_syscall:
         type: 'SYSCALL'
         syscall: 135
         a0: 40000
-    condition: selection
+    selection_path:
+        - type: 'PATH'
+        - name: '/randomize_va_space' #/proc/sys/kernel/randomize_va_space
```

this is an `or` list. You probably want an `and`?
```suggestion
    selection_path:
        type: 'PATH'
        name|endswith: '/randomize_va_space' #/proc/sys/kernel/randomize_va_space
```